app/controllers/label_templates_controller.rb

```diff
@@ -100,6 +100,15 @@ def zpl_preview
     end
   end
 
+  def sync_fluics_templates
```

Comment:
Is there supposed to be a route for this, or is it part of another ticket?

Reply:
It can be added with a button, later on